recipes/ltr_finder_parallel/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -x -e
+
+LTR_FINDER_PARALLEL_DIR=${PREFIX}/share/LTR_FINDER_parallel
+
+mkdir -p ${PREFIX}/bin
+mkdir -p ${LTR_FINDER_PARALLEL_DIR}
+cp -r * ${LTR_FINDER_PARALLEL_DIR}
+
+sed -i.bak 's|\$ltr_finder=~s/ltr_finder\\n?//;|\$ltr_finder=~s/\\bltr_finder\\n?\$//m;|' \
```

Comment:
Could you leave a comment here explaining what this line does?

Reply:
Thank you @kbseah 

I have added a comment.